cmd/minishift/cmd/start.go

```diff
@@ -353,6 +353,7 @@ func initClusterUpFlags() {
 	clusterUpFlagSet.Int(serverLogLevel, 0, "Log level for the OpenShift server.")
 	clusterUpFlagSet.StringSliceVarP(&openShiftEnv, openshiftEnv, "e", []string{}, "Specify key-value pairs of environment variables to set on the OpenShift container.")
 	clusterUpFlagSet.Bool(metrics, false, "Install metrics (experimental)")
+	clusterUpFlagSet.Bool(logging, false, "Install logging (experimental)")
```

Comment:
@hferentschik when you start it with --logging then it should start `kibana` container for log aggregation and also update master-config.yml for update url like below.

- https://docs.openshift.com/container-platform/3.3/install_config/aggregate_logging.html#aggregate-logging-kibana

```
  extensions: null
  loggingPublicURL: https://kibana-logging.192.168.42.244.nip.io
```